backend/app.py

```
@@ -27,6 +30,17 @@
 def health_check():
     return {"message": "서버가 정상 실행중입니다."}
 
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.12.2)

Unused function argument: `app`

(ARG001)